### ISSUE TITLE
Bug?

### ISSUE THREAD

**Issue by mwg3** [m4ff/UNExT#1]
Thanks for good API! Glad too see projects like this!
But i have a problem
When im calling my method
        public void StartGame()
        {
            ServerChangeScene("Game");
        }
and in method  public override void OnClientSceneChanged(NetworkConnection c)             
i'm calling ClientScene.AddPlayer(c, 0); that is not work for others player. Ofc second player is created but he doesn't has own controller and we're the same player :/
Could you help me? Thanks a lot! PS. Sorry for bad english

**Comment by m4ff** [comment]
The problem doesn't occur if you're using the base implementation of NetworkManager?

**Comment by mwg3** [comment]
Im using the base implementation. This is my own class https://pastebin.com/y4k7EqL4 and this is API from NetworkManager https://pastebin.com/bLTBuhgw
Bug is really weird, now when im in lobby with my friend and i will press the "start" button, only for me the scene changes :/
 im using ServerChangeScene..

**Comment by m4ff** [comment]
> Im using the base implementation

Sorry, I meant the actual NetworkManager class that comes with UNET.

You seem to never call the method CreateLobby() you defined. That is the method with the code to start the host. Is it possible you're calling `ServerChangeScene` before starting the host? That would explain why the scene doesn't change for the client.

**Comment by mwg3** [comment]
No, no. In the first step im using the method 'CreateLobby'. I have button for this method. When everyone is in the lobby, owner calls the method 'ServerChangeState'
